intel_extension_for_transformers/llm/runtime/graph/models/model_utils/model_utils.cpp

```diff
@@ -81,9 +81,14 @@ static bool kv_cache_init(const struct model_hparams& hparams, struct model_kv_c
   }
 
   // NE_TYPE_JBLAS can not be allocated memory
-  cache.k = ne_new_tensor_1d(cache.ctx, wtype == NE_TYPE_JBLAS ? NE_TYPE_I8 : wtype, n_elements_k, NE_SIZE_CALC);
+  const auto wtype_alloc = wtype == NE_TYPE_JBLAS ? NE_TYPE_I8;
+  cache.k = ne_new_tensor_1d(cache.ctx, wtype_alloc, n_elements_k + NE_ALIGNMENT, NE_SIZE_CALC);
```

Comment:
if 64 is what we need to align to, should it be ```(n_elements_k - NE_ALIGNMENT)/NE_ALIGNMENT*NE_ALIGNMENT```

Reply:
Well, currently there is no guarantee that it will provides aligned memory chunks. Therefore, I ask a bit more during allocation and applied an offset (ne_view_1d) to get a tensor with aligned memory.